product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/index.mdx

```diff
@@ -8,9 +8,9 @@ Monitoring integrations are configured at the project level in BigAnimal. You ca
 
 By default, all the integrations are disabled. After creating the project, enable an integration using the **Integrations** tab.
 
-All the metrics collected from all the clusters in the project are sent to the integrated tool and displayed in the BigAnimal  [**Monitoring and logging** tab using BigAnimal's Observability](../monitoring_using_biganimal_observability). The collected logs are exported to the object storage by default.
+All the metrics collected from all the clusters in the project are sent to the integrated tool and displayed in the BigAnimal **Monitoring and logging** tab [using BigAnimal's Observability](../monitoring_using_biganimal_observability). The collected logs are exported to the object storage by default.
 
-You can enable in-app inbox or email notifications to get notified incase third-party monitoring integration fails. For more information, see [managing notifications](../../../administering_cluster/notifications/#manage-notifications).
+You can enable in-app inbox or email notifications so that you are notified if third-party monitoring integration fails. For more information, see [Managing notifications](../../../administering_cluster/notifications/#managing-notifications).
```

Comment:
```suggestion
You can enable in-app inbox or email notifications so that you're notified if third-party monitoring integration fails. For more information, see [Managing notifications](../../../administering_cluster/notifications/#managing-notifications).
```